src/loader/AutoLoginImGui.cpp

```diff
@@ -0,0 +1,2010 @@
+				});
+
+			CharacterTable(search);
+
+			ImGui::EndChild();
+			ImGui::EndTabItem();
+		}
+
+		ImGui::SetNextItemWidth(tab_size);
+		if (ImGui::BeginTabItem("Settings"))
+		{
+			static auto is_paused = login::db::CacheSetting<bool>("is_paused", false, GetBoolFromString);
+			static auto debug = login::db::CacheSetting<bool>("debug", false, GetBoolFromString);
+			static auto kick_active = login::db::CacheSetting<bool>("kick_active", true, GetBoolFromString);
+			static auto end_after_select = login::db::CacheSetting<bool>("end_after_select", false, GetBoolFromString);
+			static auto load_ini = login::db::CacheSetting<bool>("load_ini", false, GetBoolFromString);
+			static auto char_select_delay = login::db::CacheSetting<int>("char_select_delay", 3, GetIntFromString);
+			static auto connect_retries = login::db::CacheSetting<int>("connect_retries", 0, GetIntFromString);
+
+			static auto password_timeout_hours = login::db::CacheSetting<int>("password_timeout_hours", 720, GetIntFromString);
+			static std::string hours_label = fmt::format("Hours to Save Password ({:.1f} days)", static_cast<float>(password_timeout_hours.Read()) / 24.f);
+			static auto perpetual_password = login::db::CacheSetting<bool>("perpetual_password", false, GetBoolFromString);
+
+			static ServerTypeInfo server_type_info;
+			static ServerNameInfo server_name_info;
+
+			if (ImGui::Checkbox("Pause on Start", &is_paused.Read()))
+				login::db::WriteSetting("is_paused", is_paused.Updated() ? "true" : "false", "Pause autologin when client starts");
+			ImGui::SetItemTooltip("Pause autologin when client starts");
+
+			ImGui::Spacing();
+			if (ImGui::Checkbox("Debug Output", &debug.Read()))
+				login::db::WriteSetting("debug", debug.Updated() ? "true" : "false", "Show plugin debug statements");
+			ImGui::SetItemTooltip("Show plugin debug statements");
+
+			ImGui::Spacing();
+			if (ImGui::Checkbox("Kick Active Character", &kick_active.Read()))
+				login::db::WriteSetting("kick_active", kick_active.Updated() ? "true" : "false", "Automatically kick the active character when prompted");
+			ImGui::SetItemTooltip("Automatically kick the active character when prompted");
+
+			ImGui::Spacing();
+			if (ImGui::Checkbox("End After Character Select", &end_after_select.Read()))
+				login::db::WriteSetting("end_after_select", end_after_select.Updated() ? "true" : "false", "Disable the plugin once the character has been selected");
+			ImGui::SetItemTooltip("Disable the plugin once the character has been selected");
+
+			ImGui::Spacing();
+			if (ImGui::Checkbox("Load Legacy Config Next Load", &load_ini.Read()))
+				login::db::WriteSetting("load_ini", load_ini.Updated() ? "true" : "false", "Import data from autologin ini file one time at load");
+			ImGui::SetItemTooltip("Import data from autologin ini file one time at load");
+			
+			ImGui::Spacing();
+			if (ImGui::InputScalar("Character Select Delay", ImGuiDataType_U32, &char_select_delay.Read()))
```

Comment:
These text boxes are way too wide. (You could also use InputInt or DragInt).

We should set a fixed width to all the elements in this pane so that it is more readable and doesn't do wierd things when stretched horizontally.